### PR TITLE
ci: add make cmd for output go tests in junit format

### PR DESCRIPTION
**Pull request by dmurray-lacework**
Signed-off-by: Darren Murray <darren.murray@lacework.net>

### CODE REVIEW

**Comment by dmurray-lacework** [comment]
update test

**Comment by dmurray-lacework** [comment]
update test

**Comment by dmurray-lacework** [comment]
update-test

**Comment by dmurray-lacework** [comment]
update test

File: Makefile [hunk 1]
```diff
@@ -169,6 +176,9 @@ endif
 ifeq (, $(shell which gotestsum))
 	GOFLAGS=-mod=readonly go install gotest.tools/gotestsum@$(GOTESTSUMVERSION)
 endif
+ifeq (, $(shell which gotestsum))
```
Comment (afiune) [inline Makefile:179]:
```suggestion
ifeq (, $(shell which go-junit-report))
```

**Review by afiune — APPROVED** [review]

**Comment by dmurray-lacework** [comment]
update test

**Pull request merged**